ophasebase/models.py

```diff
@@ -152,6 +152,8 @@ class Meta:
     name = models.CharField(max_length=100, verbose_name=_('Name'))
     description_template = models.CharField(max_length=50, verbose_name=_('Beschreibung in Template'))
     lang = models.CharField(max_length=5, verbose_name=_('Sprachcode'), default="de")
+    details_template = models.CharField(max_length=50, verbose_name=_('Template für Stundenplan View'))
```

Comment:
You could replace these two fields with a [slug](https://en.wikipedia.org/wiki/Semantic_URL#Slug) field containing an url name which could also be used as a template name. There should also be a method `getSlug()`, which uses the name of the categorie if the slug field is empty. This avoids actions requiered by the user when upgrading.